### PR TITLE
Prevent team griefing on scored matches

### PR DESCRIPTION
**Pull request by Pablete1234**
A new config option for under how many (negative) points, a player should be kicked out of the match and not allowed to rejoin. This prevents team griefing in TDM dying to the enemy team doesn't substract score so that won't affect this, only purposely suiciding will end up triggering it

image

### CODE REVIEW

**Comment by Indicardo** [comment]
I think the default -10 "grief score" is too much, as for a group of 4 that would result in -40 points

**Comment by Pablete1234** [comment]
> I think the default -10 "grief score" is too much, as for a group of 4 that would result in -40 points

While i agree it may be a bit too much, i'd rather keep the default on a number that is pretty much guaranteed not to be an issue that will affect anyone, can be tweaked/lowered for different scenarios, like public matches

**Comment by KingOfSquares** [comment]
This feels like a moderation thing and not a PGM feature. Maybe put it in community?

**Comment by Pablete1234** [comment]
> This feels like a moderation thing and not a PGM feature. Maybe put it in community?

I think this is as much moderation as preventing breaking your own core is, or tnt defusing, it doesn't issue any punishment it just prevents you from keeping on doing it

**Comment by KingOfSquares** [comment]
Then maybe a rephrasing, `You made a negative impact` sound like the plugin judging you for being bad :joy:

Maybe something more `[...]because of intentional misfit[...]` or something giving the vibe of intentional bad actions

**Comment by Pablete1234** [comment]
> Then maybe a rephrasing, `You made a negative impact` sound like the plugin judging you for being bad 
> 
> Maybe something more `[...]because of intentional misfit[...]` or something giving the vibe of intentional bad actions

Well the point is the plugin isn't 100% sure, so it's just a bit more of an open "you weren't helping your team with how you played", unsure if intentional or not

File: util/src/main/i18n/templates/join.properties [hunk 1]
```diff
@@ -35,6 +35,8 @@ join.err.noPermission = You are not allowed to join this match
 
 join.err.alreadyJoined = You have already joined the match
 
+join.err.teamGrief = You made a negative impact for your team and can no longer play this match
```
Comment (Electroid) [inline util/src/main/i18n/templates/join.properties]: You caused too much negative impact to play this match

**Review by Electroid — CHANGES_REQUESTED** [review]

**Pull request merged**